### PR TITLE
Introduce MySQL data collector

### PR DESCRIPTION
**Pull request by MaisenbacherD**
If the environment file `recipes/docker-compose/.env` exists, the JSON
output files of a benchmark run are collected into the specified MySQL
instance.

A new subcommand can collect JSON output files from old benchmarks runs
into the MySQL instance defined by the environment file.

A simple MySQL instance can be spun up by the provided docker-compose
file.

Signed-off-by: Dennis Maisenbacher <dennis.maisenbacher@wdc.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,52 @@ Run specific benchmark with the none scheduler:
...
+created and requires the same fields as in the example environment file
+`recipes/docker-compose/dev-env`.
+
+If the environment file is not present the data collection is skipped.
+
+The environment file points to a running MySQL instance, that requires the
+configuration of `recipes/docker-compose/conf.d/config-file.cnf`. A local
+MySQL instance can be started with:
+```
+docker-compose -f ./recipes/docker-compose/mongodb.yml up
```
Comment (metaspace) [inline README.md:92]: mongodb?

**Review by metaspace — CHANGES_REQUESTED** [review]

**Comment by MaisenbacherD** [comment]
Rebased this PR onto main

**Comment by MaisenbacherD** [comment]
This PR needs some rework: Make use of SQL joins

**Comment by MaisenbacherD** [comment]
Closing PR as this feature will be reworked and addressed in a new PR.

**Pull request closed without merging**